src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -372,4 +386,55 @@ public void deleteInventory(MedicalInventory medicalInventory) throws OHServiceE
 		}
 		ioOperations.deleteInventory(medicalInventory);
 	}
+
+	/**
+	 * Confirm the Inventory rows of inventory.
+	 *
+	 * @param inventory - The {@link MedicalInventory}
+	 * @param inventoryRowSearchList- The list of {@link MedicalInventory}
+	 * @throws OHDataValidationException
+	 */
+	public void confirmMedicalInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList) throws OHServiceException {
+		// validate the inventory
+		this.validateMedicalInventoryRow(inventory, inventoryRowSearchList);
+		// create Movements
+		String dischargeCode = inventory.getDischargeType();
+		String chargeCode = inventory.getChargeType();
+		Integer supplierId = inventory.getSupplier();
+		String wardCode = inventory.getDestination();
+		MovementType chargeType = medicalDsrStockMovementTypeBrowserManager.getMovementType(chargeCode);
+		MovementType dischargeType = medicalDsrStockMovementTypeBrowserManager.getMovementType(dischargeCode);
+		Supplier supplier = supplierManager.getByID(supplierId);
+		Ward ward = wardManager.findWard(wardCode);
+		LocalDateTime today = LocalDateTime.now();
+		for (Iterator<MedicalInventoryRow> iterator = inventoryRowSearchList.iterator(); iterator.hasNext();) {
+			String reference = "Ref-" + LocalDateTime.now();
```

Comment:
Make sense to prefix with "Inv-"?
```suggestion
			String reference = "Inv-" + LocalDateTime.now();
```